### DISCUSSION TITLE
Welcome to SimpleCargo Discussions!

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/SimpleCargoSolutions#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by MystLeissa** [comment]
Was wondering if you'd be willing to update this to 1.3.1 soon? I know you have other projects in the way, but this one seems useful because the Cargo Bays in the stock program for 1.25m aren't really cargo bays as much as service bay pods (meaning you can't easily store small/micro probes in them for example).

**Reply by zer0Kerbal** [reply]
am just waiting for CurseForge Support to transfer the listing over to me and then it will go boom. I will even do you one better @MystLeissa, I will update it to 1.12.3! 🤗 ;p
have ~~load~~ lots to show when this goes live.!

**Reply by zer0Kerbal** [reply]
Plus there is DaMichel's Cargo Bays... which I will be working on about the same time. Just want to power through Moho HVAC: today is Oblivion Aerospace with some really special and unique cone shaped heat shields/aeroshells - then The Art of Reentry - with some really drop-Jeb gorgeous textures.

**Reply by MystLeissa** [reply]
So Many Version Numbers to remember thanks for remembering for me...

**Reply by zer0Kerbal** [reply]
well, I have to take a break - but the repo is now setup for the next release (well, 97% there)
